pkgs/by-name/go/govers/package.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule {
+  pname = "govers";
+  version = "unstable-2016-06-23";
```

```suggestion
  version = "0-unstable-2016-06-23";
```
conventions for this have also changed. :)

---

Applied changes!